traffic_portal/app/src/common/service/utils/LocationUtils.js

```diff
@@ -19,6 +19,11 @@
 
 var LocationUtils = function($location, $uibModal) {
 
+    // ... this one allows Static DNS entries with only a trailing period
+    this.StaticDNSPattern = /^(?=.{1,253}\.?$)(?:(?!-|[^.]+_)[A-Za-z0-9-_]{1,63}(?:\.)){2,}$/;
```

Comment:
this probably makes more sense in DeliveryServiceUtils

Reply:
Ok